content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -89,29 +89,27 @@ the training process. To enable this, you will need to set up a
 Now we need to add a training stage to `dvc.yaml` including `checkpoint: true`
 in its <abbr>output</abbr>. This tells DVC which <abbr>cached</abbr> output(s)
 to use to resume the experiment later (a circular dependency). We'll do this
-with `dvc stage add`.
+with `dvc exp init`.
 
 ```dvc
-$ dvc stage add --name train \
-                --deps data/MNIST --deps train.py \
-                --params seed,lr,weight_decay \
-                --checkpoints model.pt \
-                --plots-no-cache predictions.json \
-                --plots-no-cache dvclive/scalars \
-                --metrics-no-cache dvclive.json
-                python train.py
+$ dvc exp init \
+--data 'data/MNIST' \
+--code 'train.py' \
+--live 'dvclive' \
+--model 'model.pt' \
+--type 'checkpoint' \
+python train.py
 ```
 
-The checkpoints need to be enabled in DVC at the pipeline level. The
-`-c / --checkpoint` option of the `dvc stage add` command defines the checkpoint
-file or directory. The checkpoint file, _model.pt_, is an output from one
-checkpoint that becomes a dependency for the next checkpoint, such as the model
-weights file.
+We use the `--model` and `--type` flags to define the checkpoint file or
```

Comment:
WDYT about introducing `--type checkpoint` first as the way to enable checkpoints, and then explain in the rest of the paragraph how `--model` defines the checkpoint file and what that means?